posts/announcing-exercism-research.md

```diff
@@ -0,0 +1,13 @@
+In August, we [announced our partnership](https://exercism.io/blog/redesigning-tracks-in-partnership-with-chicago-university-and-sloan-foundation) with James Evans and Gary Lupyan to carry out research into how people's programming experience affects how they learn languages.
+
+We spent some time earlier this month meeting with James and Gary and planning out how we achieve some of their wider research goals alongside the redesign of Exercism tracks I discussed in the previous post. It was clear that there are many areas where Exercism will be able to help their research but that those might not fit directly into core Exercism product. We have therefore decided to create a new area of Exercism specifically centred around research, called (unimaginatively) Exercism Research. This subsite will host Experiments that aim to gather data on a specific research goal. The experiments will also have the added benefit of allowing us to test out new technology on a less critical area of the site.
+
+The first experiment, which we aim to launch just after Christmas, will be centred around investigating how someone's previous programming experience impacts the reusability of the code they produce in different languages. The experiment will be structured as followed:
```

Comment:
> which we aim to launch just after Christmas

Can we say "just after New Year's" instead? I know Christmas is pretty secular in a lot of places, but in the US it's a very religious thing. (I know not all places have new years at the same time, but I'm ok with being western-centric on that point).

Reply:
I've gone for "in the first week of January"